diff -urN anubis-4.0.org/configure.ac anubis-4.0/configure.ac
--- anubis-4.0.org/configure.ac	2004-12-18 12:10:03.000000000 +0100
+++ anubis-4.0/configure.ac	2005-02-12 20:38:18.000000000 +0100
@@ -95,7 +95,6 @@
 ])
 
 AC_CHECK_LIB(socket, socket)
-AC_CHECK_LIB(nsl, gethostbyaddr)
 
 AC_SUBST(INCLUDES)
 case $build in
@@ -397,7 +396,6 @@
 	[with_tcp_wrappers=no])
 if test "$with_tcp_wrappers" = "yes"; then
  AC_CHECK_LIB(wrap, main,, with_tcp_wrappers=no)
- AC_CHECK_LIB(nsl, main,, with_tcp_wrappers=no)
  AC_CHECK_HEADERS(tcpd.h,, with_tcp_wrappers=no)
 fi
 if test "$with_tcp_wrappers" = "yes"; then
