--- anubis-3.0.1/configure.in~	2002-06-08 23:04:40.000000000 +0200
+++ anubis-3.0.1/configure.in	2002-06-09 10:59:51.000000000 +0200
@@ -46,8 +46,10 @@
 AC_CHECK_HEADERS(string.h arpa/inet.h syslog.h regex.h)
 
 dnl Check for getopt_long in standard library
-AC_SUBST(LIBOBJS)
-AC_CHECK_FUNCS(getopt_long,, [LIBOBJS="$LIBOBJS getopt1.o getopt.o"])
+AC_CHECK_FUNCS(getopt_long,, [
+	AC_LIBOBJ(getopt1)
+	AC_LIBOBJ(getopt)
+])
 
 dnl Checks for library functions.
 AC_CHECK_FUNCS(snprintf vsnprintf syslog setrlimit socketpair strdup)
